pkg/monitoring/rules/rules.go

```diff
@@ -1,180 +1,56 @@
 package rules
 
 import (
-	"fmt"
-
+	"github.com/machadovilaca/operator-observability/pkg/operatorrules"
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
-	"k8s.io/apimachinery/pkg/util/intstr"
-	"k8s.io/utils/ptr"
-)
 
-const (
-	severityAlertLabelKey     = "severity"
-	healthImpactAlertLabelKey = "operator_health_impact"
-	partOfAlertLabelKey       = "kubernetes_operator_part_of"
-	partOfAlertLabelValue     = "kubevirt"
-	componentAlertLabelKey    = "kubernetes_operator_component"
-	componentAlertLabelValue  = "ssp-operator"
+	"kubevirt.io/ssp-operator/pkg/monitoring/rules/alerts"
+	"kubevirt.io/ssp-operator/pkg/monitoring/rules/recordingrules"
 )
 
 const (
-	CommonTemplatesRestoredIncreaseQuery   = "sum(increase(kubevirt_ssp_common_templates_restored_total{pod=~'ssp-operator.*'}[1h]))"
-	TemplateValidatorRejectedIncreaseQuery = "sum(increase(kubevirt_ssp_template_validator_rejected_total{pod=~'virt-template-validator.*'}[1h]))"
+	ruleName             = "prometheus-k8s-rules-cnv"
```

Comment:
This same constant is also defined in `internal/operands/metrics/resources.go` as `PrometheusRuleName`, can you remove one of these constants?

Reply:
I can't do that since it's making  "import cycle not allowed"

Reply:
I'm not getting an import cycle. But nevermind, I will post a new PR later.

Reply:
Can you use the constant from this file in the other places? I've tried it and I'm not getting an import cycle.